src/components/checkbox/checkbox.scss

```diff
@@ -96,10 +96,9 @@
 
   &.carbon-checkbox__help-text--inline {
     display: inline;
-    left: 21px;
-    margin-left: -21px;
-    margin-right: 5px;
+    left: 0;
```

Comment:
![screen shot 2018-01-19 at 15 33 28](https://user-images.githubusercontent.com/1668080/35158146-21631a42-fd2e-11e7-994d-8f671ec89430.png)

setting `left: 0` and removing `margin-left` breaks the non-reversed version. Looks like it works if you remove `left` and set `margin-left` to `0`.

Reply:
Maybe I am blind (but I can't see what broke).I removed both. We should need either. right?

Reply:
The gap between the icon and the help text is too wide

Reply:
`margin-left: 0` ensures it is correct